.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1367,7 +1466,7 @@ stages:
 
 - stage: dotnet_tool
   condition: and(succeeded(), eq(variables['isScheduledBuild'], 'False'), ne(variables['isNgenTestBuild'], 'True'))
-  dependsOn: [build_windows_tracer, build_linux, build_arm64, build_macos, master_commit_id]
+  dependsOn: [build_windows_tracer, build_linux_tracer, build_arm64, build_macos, master_commit_id]
```

Comment:
This step is not correct, the windows case did not include the profiler/native loader.

Reply:
This step will depend on the `package linux` and `package windows` stages if we want to go that way.

Reply:
As discussed, we'll leave this change for a later PR.

Reply:
> I wonder if it makes sense to do this sooner rather than later 😄 Currently some customers will be using the native loader in production, and some will not... The same goes for the windows-tracer-home

I can if you think it's mandatory, but to me it does not belong to this PR and it will clutter it as well.
We can do it soon in a later PR :P

Reply:
Agree that we _must_ ship both changes together.